cumulusci/cli/cci.py

```diff
@@ -581,29 +590,38 @@ def service_list(config):
 
 
 class ConnectServiceCommand(click.MultiCommand):
+    load_config_kwargs = {"allow_global_keychain": True}
+
+    def _get_services_config(self, config):
+        return (
+            config.project_config.services
+            if not config.global_keychain
+            else config.global_config.services
+        )
+
     def list_commands(self, ctx):
         """ list the services that can be configured """
-        config = load_config()
-        return sorted(config.project_config.services.keys())
+        config = load_config(**self.load_config_kwargs)
```

Comment:
load_config is called in two places in that class and should always have the same kwargs.  It seemed better to set it in one place and use that dict.

Reply:
How about define

    def _load_config(self):
        return load_config(allow_global_keychain=True)

and use that in both places?

This is probably mostly a matter of taste and I'll defer to your preference, but I think it makes the code easier to read because you don't have to go find where self.load_config_kwargs is defined or search the codebase to check whether it's getting set from somewhere else.